ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -249,22 +327,26 @@ by maximizing $\Sigma_{\forall i} (g_L - g_U)_i$ to maximize $x_W - x_A$.
 > [!TIP]
 > Although $D$ shadows a variable in ACP-176, that one is very different to anything here so there won't be confusion.
 
-Recall that the proportionality $p$ of capacity per second and target gas per second ($R = pT$) results in increasing excess such that
+Recall that the increasing excess occurs such that
 
 $$
-x := x + \frac{g (p-1)}{p}.
+x := x + g \cdot \frac{(R - T)}{R}
 $$
 
-If we allow the queue to have a bandwidth of $R\lambda^{-1}$ gas/second and limit its length $\tau$ seconds or a back-pressure window of $\omega = \tau R \lambda^{-1}$ gas, we can derive an upper bound on the difference in the changes to worst-case and actual gas excess:
+Since we limit the size of the queue to $\omega$, we can derive an upper bound on the difference in the changes to worst-case and actual gas excess:
```

Comment:
```suggestion
Since we limit the size of the queue to $\omega$, we can derive an upper bound on the difference in the changes to worst-case and actual gas excess caused by the transactions in the queue:
```